src/components/object-info/ObjectInfo.jsx

```diff
@@ -16,6 +16,17 @@ const objectInspectorTheme = {
 }
 
 // TODO: Use https://github.com/multiformats/multicodec/blob/master/table.csv to get full name.
+/**
+ * Note that existing colors were added into the "Mass Editor" at colordesigner.io:
+ *
+ * * #28CA9F
+ * * #244e66
+ * * #378085
+ * * #f14e32
+ * * #383838
+ *
+ * And then hamt-sharded-directory was set to #9f28ca which was listed under "Color Harmonies - Triad" on 2023-05-22
+ */
 const nodeStyles = {
   'dag-cbor': { shortName: 'CBOR', name: 'dag-cbor', color: '#28CA9F' },
```

Comment:
@SgtPooki thoughts on opportunistic color  cleanup here? 

We are wasting good color on GIT, let's use better one here, as CBOR is way more useful
```suggestion
  'dag-cbor': { shortName: 'CBOR', name: 'dag-cbor', color: '#378085' },
```

(and set `git-raw` to `#383838` -- same as ETH block which we will be removing)


The end result would be easier on the eyes:

> ![2023-05-26_13-53](https://github.com/ipfs/ipld-explorer-components/assets/157609/95af83f3-14c8-4fa6-ad62-759fd80c30c0)

Reply:
I’m good with this. I thought about changing the whole palette, even using ipfs-css to get appropriately accessible colors similar to our other tooling. Still, I wasn't sure whether to make that change, given all the other changes in this PR.